pkg/verifier/aws/entry_point.go

```diff
@@ -61,6 +61,7 @@ func (a *AwsVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 		"VALIDATOR_START_VERIFIER": "VALIDATOR START",
 		"VALIDATOR_END_VERIFIER":   "VALIDATOR END",
 		"VALIDATOR_IMAGE":          networkValidatorImage,
+		"VALIDATOR_REPO":           networkValidatorRepo,
```

Comment:
This wasn't being set in the past and must have slipped through :D

Reply:
This should be calculated through VALIDATOR_REFERENCE rather than duplicating entries.